### ISSUE TITLE
nearText documentation is hard to find for Get

### ISSUE THREAD

**Issue by raam86** [semi-technologies/weaviate-io#89]
At the moment the nearText stanza is documented under https://weaviate.io/developers/weaviate/current/retriever-vectorizer-modules/text2vec-transformers.html#additional-graphql-api-parameters

I would expect to find the full reference under Get graphql https://weaviate.io/developers/weaviate/current/graphql-references/get.html

**Comment by etiennedi** [comment]
Yeah, I think we could add a list of all possible arguments to the Get page with links to the respective modules. E.g. `nearText` linking to text2vec-contextionary and text2vec-transformers, `nearImage` linking to clip and img2vec, etc.

I think they are also hard to find because they are (also) on a page labelled "Filters", but they are not filters, but search operators, see also #34

**Comment by etiennedi** [comment]
I'm just working on this page anyway, I'll take care of this.

**Comment by etiennedi** [comment]
Closing as complete.
